app/renderer/js/main.js

```diff
@@ -142,7 +144,11 @@ class ServerManagerView {
 
 		for (const i in settingOptions) {
 			if (ConfigUtil.getConfigItem(i) === null) {
-				ConfigUtil.setConfigItem(i, settingOptions[i]);
+				if (EnterpriseUtil.isConfigItemExists(i)) {
```

Comment:
The `i` here is a setting name here right? I think it makes sense to change the `i` variable `setting` in a separate commit.